build/Targets/VSL.Versions.targets

```diff
@@ -61,14 +61,13 @@
 
   <!-- NuGet version -->
   <PropertyGroup>
-    <VSSemanticVersion>15.0.1</VSSemanticVersion>
+    <VSSemanticVersion>15.0</VSSemanticVersion>
```

Comment:
The name makes it sound generic. But VSSemanticVersion is used just for Nuget version and not outside.